### ISSUE TITLE
Sprint w/c 6 March 2023

### ISSUE THREAD

**Issue by rufuspollock** [datopian/datahub-next#25]
- [ ] Rendering all our local content through our "MDX+D" pipeline
  - [x] #31 
  - [ ] #27 **🚧2023-03-12 90% was v nearly there on Friday. Seems to be blocked on an 🐛 **
  - [ ] New content layer / contentbase implementation **🚧2023-03-13 50% of there with #31 - need to actually replace now**
- [x] Improved table presentation e.g. filtering, sorting, pagination (or infinite scroll?) #29 **✅2023-03-07 deployed**
 
🆕  in sprint

- [x] #4 - got the SSR build working on vercel
- [x] #3 - that plus table work meant we were done

BONUS

- [ ] Processing remote content e.g. from github.com/datasets/awesome-data/issues **🚧2023-03-13 not processing the content but having a working live remote processing. See #3**

Architecture

- [x] Product vision **✅2023-03-13 had quite a bit of thinking on this and a good discussion. Definite focus on Scratch approach leading into the Project ...**
- [x] Mockups of ...? **✅2023-03-13 ❌ did not really do any mockups in the end.**
- [x] 🆕  Architecture diagram **✅2023-03-13 see https://datahub.io/notes/design**

# Sprint Review

Agenda

- [x] 🧘 Meditation
- [x] Standup and blockers
- [x] Sprint review
- [x] AOB **✅2023-03-12 moved to https://github.com/datopian/datahub-next/issues/26**

Next steps ⏭️

- [x] Add minor items above to inbox @rufuspollock **✅2023-03-12 moved to https://github.com/datopian/datahub-next/issues/26**
- [ ] @olayway to open issue on flowershow with things that could be cleaner based on experience doing upgrade here
- [ ] @olayway extract notes on issues with edge and flowershow/next-mdx-remote/everything into a notes/nextjs-edge-learnings

## Closing out sprint

- [x] https://github.com/datopian/datahub-next/issues/20 - data table analysis :+1: **✅2023-03-12**
  - [x] how does this merge to the main page? **💬2023-03-10 just copy it across (bonus for small integration)**
  - [x] The ranking in spreadsheet can probably go into the issue itself or perhaps in an issue design-data-table?
- [ ] Architecture @rufuspollock 
  - [ ] 🔽 Notes re ContentBase and MarkdownDB
    - [ ] Notes on sanity, notion and obsidian linked from contentbase
    - [ ] Notes on sanity
- [ ] https://github.com/datopian/datahub-next/issues/27 :+1: 
  - [x] ola rebase her pipeline branch off main (and it is ok to do SSG for the demo/test file)
- [ ] Deploy item
  - [x] @joao: reiterate doubt that symlink issue really is an issue - would be nice to not make this change (unless we backport to main) b/c it will be a pain on every merge across if we keep two branches **✅2023-03-12 joao created a separate PR that fixes this**
  - [ ] @joao: only build ssr branch on vercel for now - see https://vercel.com/guides/how-do-i-use-the-ignored-build-step-field-on-vercel

**Comment by rufuspollock** [comment]
FIXED. Would say 70-80% result plus some extras.
